### PR TITLE
Cleanup sharing leftovers

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

File: apps/files_sharing/src/utils/SharedWithMe.js [hunk 1]
```diff
@@ -27,7 +27,7 @@
  * @returns {string} the title
  */
 const shareWithTitle = function(share) {
-	if (share.type === OC.Share.type_GROUP) {
+	if (share.type === OC.Share.SHARE_TYPE_GROUP) {
```
Comment (skjnldsv) [inline apps/files_sharing/src/utils/SharedWithMe.js]: Somehow I missed this :man_shrugging:

**Comment by skjnldsv** [comment]
I forgot some scripts and I'm not really happy with how it's design, especially the way we declare th OC.Sharing.SHARE_TYPE_XXX constants.

**Review by ChristophWurst — APPROVED** [review]
:elephant:

**Review by rullzer — APPROVED** [review]
Did some smoke testing. Didn't do :boom:
Lets do this.

**Pull request merged**